code/datums/actions/mobs/personality_commune.dm

```diff
@@ -0,0 +1,53 @@
+// This can probably be changed to use mind linker at some point
+/datum/action/personality_commune
+	name = "Personality Commune"
+	desc = "Sends thoughts to your alternate consciousness."
+	background_icon_state = "bg_spell"
+	button_icon = 'icons/mob/actions/actions_spells.dmi'
+	button_icon_state = "telepathy"
+	overlay_icon_state = "bg_spell_border"
+
+	/// Fluff text shown when a message is sent to the pair
+	var/fluff_text = span_boldnotice("You hear an echoing voice in the back of your head...")
+
+/datum/action/personality_commune/New(Target)
+	. = ..()
+	if(!istype(target, /datum/brain_trauma/severe/split_personality))
+		stack_trace("[type] was created on a target that isn't a /datum/brain_trauma/severe/split_personality, this doesn't work.")
+		qdel(src)
+
+/datum/action/personality_commune/Trigger(trigger_flags)
+	. = ..()
+	if(!.)
+		return FALSE
+
+	var/datum/brain_trauma/severe/split_personality/trauma = target
+	if(!istype(trauma))
+		CRASH("Personality Commune was triggered on a target that isn't a /datum/brain_trauma/severe/split_personality.")
```

Comment:
do we really need to check this every time after we did this in `New()`

Reply:
Eh fair